web-client/terraform/main/setup_dynamsoft.sh

```diff
@@ -26,4 +26,4 @@ sudo sed -i "s|DYNAMSOFT_RESOURCE_PATH|${dynamsoft_url}/Dynamic%20Web%20TWAIN%20
 sudo sed -i "s|DYNAMSOFT_PRODUCT_KEYS|${dynamsoft_product_keys}|" /tmp/dynamsoft.webtwain.config.js.tpl
 
 # copy back from /tmp to main nginx html folder
-sudo cp /tmp/dynamsoft.webtwain.config.js.tpl Dynamic\ Web\ TWAIN\ SDK\ 17.2.5/Resources/dynamsoft.webtwain.config.js
\ No newline at end of file
+sudo cp /tmp/dynamsoft.webtwain.config.js.tpl Dynamic\ Web\ TWAIN\ SDK\ 17.2.5/Resources/dynamsoft.webtwain.config.js
```

Comment:
was a new line character removed from this .sh script?  I think all .sh scripts should end with a blank line